### ISSUE TITLE
Ubuntu not seeing the digit

### ISSUE THREAD

**Issue by stoic-roboticist** [facebookresearch/digit-interface#11]
Hi, 

I bought two digit sensors and it seems I just cant make any of my PCs to see them. When I connect them, their light comes on however when I try to use lsusb to see if they are connected nothing shows up. I tried 3 different ubuntu 20.04 machines and none can detect them. I added the supplied udev rules as per your instruction and still nothing seems to change. My only suspicion is that I need a USB 3.0 cable but to my understanding there are no USB 3.0 cables with micro B male interface. Am I missing something. Thanks for your help in advance.

**Comment by AlphaBetaPhi** [comment]
Hi @stoic-roboticist,

Thank you for your interest in the project. 

Did you reload the rules? Restart the system? What is the output of lsusb?

An USB2.0 cable is required. Is this cable you are using a data cable?

**Comment by stoic-roboticist** [comment]
Hi,

I did reload the rules and restarted my pc but still to no avail. I even used dmseg to monitor the system events and when connecting the digit, it didn't register anything. The output of my lsusb is just the different USB devices I have connected to my pc with nothing related to the digit. I tried to connect it off and on and move it around ports, still it is not being detected by any of my pcs. 
As for the cable I used two different USB2.0 cables (USB A male to USB B Micro male) and nothing seems to make any of my systems detect the sensor. is there a way to force the sensor into some other mode that will make it visible to my system?
